### DISCUSSION TITLE
Lineage Ingestion

### DISCUSSION THREAD

**Discussion by luckyliush** [open-metadata/OpenMetadata#17047] [Ideas]
Will there be automatic ingestion of Hive or Doris's bloodline in the future?

**Comment by Prajwal214** [comment]
Automated lineage for Hive is not supported, and there is no specific timeline for its release. You can use Spline to gather lineage from Hive jobs. Check [this GitHub issue](https://github.com/open-metadata/OpenMetadata/issues/12376) for more details

**Reply by luckyliush** [reply]
Thank you
